integration-test/kubernetes-api-java/pom.xml

```diff
@@ -18,9 +18,9 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <encoding>UTF-8</encoding>
-    <akka.version>2.5.19</akka.version>
+    <akka.version>2.5.27</akka.version>
```

Comment:
travis failed with "Current version of Akka is [2.5.19], but cluster-bootstrap requires version [2.5.27]", so updated this. Not sure if there are more places but didn't find anything more when searching.